pkg/cable/wireguard/driver.go

```diff
@@ -246,6 +254,13 @@ func (w *wireguard) ConnectToEndpoint(endpointInfo *natdiscovery.NATEndpointInfo
 	klog.V(log.DEBUG).Infof("Adding connection for cluster %s, %v", remoteEndpoint.Spec.ClusterID, connection)
 	w.connections[remoteEndpoint.Spec.ClusterID] = connection
 
+	port, err := remoteEndpoint.Spec.GetBackendPort(v1.UDPPortBackendConfig, int32(w.spec.NATTPort))
+	if err != nil {
+		return "", errors.Wrapf(err, "error reading UDP port")
```

Comment:
I assume we want the same behavior here as for libreswan.

```suggestion
		klog.Warningf("Error parsing %q from remote endpoint %q - using port %s instead: %v", subv1.UDPPortBackendConfig, remoteEndpoint.Spec.CableName,
			w.spec.NATTPort, err)
```

Reply:
Done.